app/router.js

```diff
@@ -17,10 +18,32 @@ const Router = EmberRouter.extend({
 
     _trackPage() {
         run.scheduleOnce('afterRender', this, () => {
+            // Tracks page with custom parameters
+            // authenticated => if the user is logged in or not
+            // isPublic      => if the resource the user is viewing is public or private.
+            //                  n/a is used for pages like discover and index
+            // page          => the name of the current page
+            // resource      => what resource the user is on. Ex node, preprint, registry.
+            // title         => the current route name
+            const {
+                authenticated,
+                isPublic,
+                resource,
+            } = config.metricsAdapters[0].dimensions;
+
             const page = document.location.pathname;
             const title = this.getWithDefault('currentRouteName', 'unknown');
+            const isAuthenticated = this.get('session.isAuthenticated');
+            const publicType = title.endsWith('content.index') ? 'public' : 'n/a';
```

Comment:
In theory you can have a private node or an unpublished preprint. This doesn't seem to handle that at all. Should it?

Reply:
After talking it out this seems like it isn't something we should worry about, since it'll be fixed after the node/preprint divorce.